README.md

```diff
@@ -76,24 +77,25 @@ If you would like to customize or prettify the archive name, page names, or the
 Manifest file should start with single Node and should have properties described 
 below (it will be displayed as type "group"):
 
-| Prop                    | Type                     | Description                                                                                                                   |
-|-------------------------|--------------------------|-------------------------------------------------------------------------------------------------------------------------------|
-| label (**required**)    | string                   | Label that will be displayed on UI for Node                                                                                   |
-| children (**required**) | Node[]                   | List of Nodes (nested groups or markdown files)                                                                               |
-| type                    |  | Can be skipped - RedisInsight will always use the "group" parameter                                                                            |
-| author                  | string                   | Not required, Author name                                                                                                                   |
-| URL                     | string                   | Not required, link to repository/web-site                                                                                                   |
-| industry                | string[]                 | Not required, tags to show relevant industry                                                                                                |
-| description             | string                   | Not required, short description of tutorials                                                                                                |
+| Prop                    | Type     | Description                                                         |
+|-------------------------|----------|---------------------------------------------------------------------|
+| label (**required**)    | string   | Label that will be displayed on UI for Node                         |
+| children (**required**) | Node[]   | List of Nodes (nested groups or markdown files)                     |
+| type                    |          | Can be skipped - RedisInsight will always use the "group" parameter |
+| author                  | string   | Not required, Author name                                           |
+| URL                     | string   | Not required, link to repository/web-site                           |
+| industry                | string[] | Not required, tags to show relevant industry                        |
+| description             | string   | Not required, short description of tutorials                        |
 
```

Comment:
please add information on what fields are displayed in RedisInsight and what are not